src/app/inventory/store/stats.ts

```diff
@@ -98,9 +98,9 @@ type StatLookup = { [statHash: number]: DimStat | undefined };
 
 /** Build the full list of stats for an item. If the item has no stats, this returns null. */
 export function buildStats(
+  defs: D2ManifestDefinitions,
   createdItem: DimItem,
-  itemDef: DestinyInventoryItemDefinition,
-  defs: D2ManifestDefinitions
+  itemDef = defs.InventoryItem.get(createdItem.hash)
```

Comment:
any reason to retain this param when it can be so quickly re-derived? feels ripe for cleanup

Reply:
Not really, it's just already there for most invocations.